features/bevragen/persoon/verblijfplaats/dev/Informatievragen-fields-gba.feature

```diff
@@ -44,46 +45,58 @@ Functionaliteit: vertalen van gevraagde samengestelde of afgeleide verblijfplaat
       | .straat              | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
       | .locatiebeschrijving | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
       |                      | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
+      | .straat              | .           | .          | .        | .          | .                                | .                             |                              |
```

Comment:
is het nodig om standaardwaarde voorbeelden op te nemen? In de onbekend_waardes_gba.feature wordt dit in principe al gezegd

Reply:
is het niet handiger om voor de vier verblijfplaats types een aparte scenario te definieren? Wat in principe varieert is de veld waarde. Wat bijv. niet in voorbeelden is opgenomen zijn de volgende verblijfplaats velden:
- verblijfplaats (de hele categorie)
- verblijfplaats.datumVan (veld op hoofd/root niveau)
- verblijfplaats.datumIngangGeldigheid,verblijfplaats.verblijfadres (meerdere verblijfplaats velden)
- verblijfplaats.verblijfadres.regel1 (veld van verblijfplaats buitenland voor een verblijfplaats binnenland als idg een object met alleen type veld moet worden geleverd)
- verblijfplaats.type

Reply:
> Dit is toch een gba-feature. Die kent geen types.

Als een consumer op proxy niveau met fields verblijfplaats.type vraag, dan wordt dit toch doorgekopt naar de gba-api?